guides/source/active_record_migrations.md

```diff
@@ -1224,17 +1487,27 @@ destroyed. Like anything which operates at the application level, these cannot
 guarantee referential integrity and so some people augment them with [foreign
 key constraints][] in the database.
 
-Although Active Record does not provide all the tools for working directly with
-such features, the `execute` method can be used to execute arbitrary SQL.
+In practice, foreign key constraints and unique indexes are generally considered
+safer when enforced at the database level. Although Active Record does not
+provide direct support for working with these database-level features, you can
+still use the execute method to run arbitrary SQL commands.
```

Comment:
```suggestion
still use the execute methods to run arbitrary SQL commands.
```

Reply:
I'm not sure that I understand the suggested change "method" to "methods" here? Maybe you can expand on it please? I'm referring to the execute method specifically. i.e. https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/DatabaseStatements.html#method-i-execute 

Side note: I could probably link to it here as well like I do above.